pyproject.toml

```diff
@@ -150,6 +150,18 @@ version = { attr = "integreat_compass.__version__" }
 [tool.setuptools.packages.find]
 include = ["integreat_compass*"]
 
+[tool.bumpver]
```

Comment:
Did you forget to add `bumpver` as dev dependency?

Reply:
Yes, I had. It's added now